### PR TITLE
fix(tools): different condition

### PR DESCRIPTION
**Pull request by naomi-lgbt**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

As @ShaunSHamilton noticed, the `branches-ignore` flag wasn't working. Believe this is because that refers to the target branch, which is `main`, not the origin branch.

This approach should work instead, going to test on a separate repo to confirm.

### CODE REVIEW

**Comment by nhcarrigan** [comment]
https://github.com/nhcarrigan/temp/pulls seems to work as intended

**Review by ShaunSHamilton — APPROVED** [review]
Sure 👍 ... I am impressed you created a fully-fledged repo with Ts&Cs.

File: .github/workflows/codesee-arch-diagram.yml [hunk 1]
```diff
@@ -4,14 +4,12 @@ on:
...
     types: [opened, synchronize, reopened]
-    branches-ignore:
-      - 'renovate/**'
-      - 'i18n-sync-**'
 
 name: CodeSee Map
 
 jobs:
   test_map_action:
+    if: ${{ github.actor != 'renovate[bot]' && github.actor != 'camperbot' }}
```
Comment (ojeytonwilliams) [inline .github/workflows/codesee-arch-diagram.yml:12]: Neat.  How did you know/figure out that it was `renovate[bot]` not just `renovate`?

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

File: .github/workflows/codesee-arch-diagram.yml (hunk 1 above)
Comment (ShaunSHamilton) [reply .github/workflows/codesee-arch-diagram.yml:12]: https://github.com/freeCodeCamp/freeCodeCamp/blob/b5019000bbfe12dfccfc144293d69285e237d756/.github/workflows/codeql-analysis.yml#L13

Example 🤷‍♂️

Comment (ojeytonwilliams) [reply .github/workflows/codesee-arch-diagram.yml:12]: Oh.  Well.  Fine. :man_facepalming:

Comment (nhcarrigan) [reply .github/workflows/codesee-arch-diagram.yml:12]: Checked my own custom github tooling lol 

(To get it initially for that I queried the API and looked at the value)

Comment (ojeytonwilliams) [reply .github/workflows/codesee-arch-diagram.yml:12]: > queried the API 

So, what, make a query like [this](https://docs.github.com/en/rest/reference/pulls#get-a-pull-request) and just look at the login?

**Comment by ShaunSHamilton** [comment]
Why did this not auto merge?

**Comment by nhcarrigan** [comment]
Oliver's conversation wasn't resolved 🙂

**Comment by ojeytonwilliams** [comment]
> Oliver's conversation wasn't resolved 🙂

Oops. Sorry about that.

**Pull request merged**